### DISCUSSION TITLE
Installation of QField Cloud on private Server

### DISCUSSION THREAD

**Discussion by meyerlor** [opengisch/QField#5188] [Q&A]
We as a company really need QField Cloud on our own server. After contacting the sales team of Qfield, we are now on a list and we will be noticed as soon they offer this as a service - we can not wait longer, so my question: can anyone support us in setting this up (ofc we will pay for it!) Please contact me if you feel up to it! :)
Lorenz

**Comment by ZsanettMed** [comment]
Hey @meyerlor, thank you for your interest. I've sent you an email to the following address: m\*r@gis\*er.de". Let's continue our conversation via email and close this discussion here.
Best, Zsanett
